overlord/servicestate/servicestate.go

```diff
@@ -43,8 +43,10 @@ import (
 )
 
 type Instruction struct {
-	Action string   `json:"action"`
-	Names  []string `json:"names"`
+	Action          string   `json:"action"`
+	Names           []string `json:"names"`
+	Scope           []string `json:"scope"`
+	ServicesOfUsers []string `json:"user-services-of"`
```

Comment:
the intention of the spec was not make this just a list, but at least at the level of the API a value that can be either a string or a list, maybe that's too annoying to implement, though a type with dedicated marshaling/unmarshling could make this work. I commented elsewhere on options here

Reply:
Yes I originally completely missed this part, I have updated the code to deal with this